README.md

```diff
@@ -142,5 +157,21 @@ Using the same front-end application and Rails API from the previous question, h
 If the Post creation is successful, the new Post should be printed to the browser console. Otherwise, an error message should be printed to the console.
 
 ```js
-// Your answer goes here...
+$.ajax({
+   url: url,
+   type: "get",
+   dataType: "json"
+   // $.ajax takes an object as an argument with at least three key-value pairs...
```

Comment:
You need a `data` attribute here if you're going to be POSTing something. How else are you gonna tell the API what exactly you're creating?